src/components/Markdown.tsx

```diff
@@ -1,40 +1,26 @@
-import ReactMarkdown from "react-markdown";
+import ReactMarkdown, { PluggableList } from "react-markdown";
 import remarkGfm from "remark-gfm";
 import rehypeRaw from "rehype-raw";
-import rehypeSanitize, { Options } from "rehype-sanitize";
+import rehypeSanitize from "rehype-sanitize";
 import { useMemo } from "react";
-import { PluggableList } from "unified";
-
 export interface MarkdownProps {
   /** Stringified Github-Flavored Markdown. */
   content: string;
-  /** HTML Sanitization Schema for use with Rehype. */
-  sanitizationSchema?: Options;
 }
 
 /**
  * Renders Github-Flavored Markdown from the Knowledge Graph. This Markdown can include
- * arbitrary HTML. Any HTML will be sanitized according to the provided Rehype Schema. If
- * no Schema is provided, Rehype's default will be used.
+ * arbitrary HTML. Any HTML will be sanitized according to Rehype's default Schema.
  */
-export function Markdown({ content, sanitizationSchema }: MarkdownProps) {
+export function Markdown({ content }: MarkdownProps) {
   const plugins = useMemo(() => {
```

Comment:
Now that computing these plugins don't depend on any props, can we move this variable outside the component?